app/models/application_record.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationRecord < ActiveRecord::Base
+  self.abstract_class = true
+  
+  def count_votes
+    return self.votes.length
+  end
```

Comment:
I like that you put this method in `ApplicationRecord` - good use of inheritance.